doc/conf.py

```diff
@@ -65,6 +65,7 @@
     'sphinx.ext.mathjax',
     'sphinx.ext.napoleon',
     'sphinxcontrib.bibtex',
+    'sphinxcontrib.plantuml', # https://github.com/sphinx-contrib/plantuml
```

Comment:
Please remove commented out entry.

```suggestion
    'sphinxcontrib.plantuml',
```

Reply:
Removed!

Reply:
Thanks, @benkozi!